torch/_inductor/triton_heuristics.py

```diff
@@ -168,6 +168,15 @@ def launcher({', '.join(def_args)}, grid, stream):
 
         return launcher
 
+    def round_benched_timing(self, x, decimal_places=1):
+        """
+        Sometimes different autotuning configs can have very small runtime
+        difference. This can affect the ranking of different configs, making
+        autotuning nondeterministic.  This function rounds the benched timing to
+        a default value of one decimal place, i.e., precision of 0.1 ms.
+        """
+        return builtins.round(x, decimal_places)
```

Comment:
Rounding could still be non-determinstic (just less likely).  Perhaps we should change the cache format to just store which one was selected?